code/datums/keybinding/human.dm

```diff
@@ -10,15 +10,6 @@
 	description = ""
 	keybind_signal = COMSIG_KB_QUICKEQUIP
 
-
-/datum/keybinding/human/holster
-	hotkey_keys = list("H")
-	name = "holster"
-	full_name = "Holster"
-	description = ""
-	keybind_signal = COMSIG_KB_HOLSTER
```

Comment:
don't remove, it will be used again when someone fixes holsters

Reply:
you want to keep a non-functionning keybind, that takes 10 seconds to remake? 

People will just keybind it and think it's a bug, which it is

Reply:
keeps people from having to rebind it if they had something else bound.